CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 
 #### Dependency Upgrade
 * Bump Knative model to v0.34.0
+* Update chaos-mesh extension to v2.1.3. Add PodHttpChaos, GCPChaos, BlockChaos and PhysicalMachineChaos.
```

Comment:
Apparently the capitalization of some generated class names has changed. We should add a note about this or revert this change.